WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -1,12 +1,29 @@
 import UIKit
 import WordPressAuthenticator
+import Combine
 
 private struct PrepublishingOption {
     let id: PrepublishingIdentifier
     let title: String
+    let type: PrepublishingCellType
 }
 
-private enum PrepublishingIdentifier {
+private enum PrepublishingCellType {
+    case value
+    case textField
+
+    var cellType: UITableViewCell.Type {
+        switch self {
+        case .value:
+            return WPTableViewCell.self
+        case .textField:
+            return WPTextFieldTableViewCell.self
```

Comment:
The dynamic type is different between `WPTextFieldTableViewCell` and `WPTableViewCell`:

<img src="https://user-images.githubusercontent.com/7040243/108556808-5d5ee500-72d6-11eb-9e35-38da8e968e2d.png" width="300">

I'd recommend making them equal because we recently received some support requests from issues associated with big font sizes.